obspy/clients/eida/__init__.py

```diff
@@ -10,120 +13,182 @@
 :license:
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
+
+Basic Usage
+-----------
+
+The first step is always to initialize a client object.
+
+>>> from obspy.clients.eida import Client
+>>> client = Client("GFZ")
+
+A client object can be initialized either with the base URL of any EIDA routing
+service or with a shortcut name that will be mapped to such a URL. The examples
```

Comment:
This is using the short URLs defined in `fdsn/header.py`, right? Would be good to have a link in the docstring, like `... :mod:`a shortcut name that will be mapped to such a URL <obspy.clients.fdsn>` ...`